app/src/androidTest/java/com/github/gogetters/letsgo/GameActivityTest.kt

```diff
@@ -25,9 +25,8 @@ class GameActivityTest {
     fun tappingScreenPlacesStone() {
         val scenario = activityScenarioRule.scenario
         val goView = Espresso.onView(withParent(withId(R.id.gameBoardFrame)))
-        //goView.perform(touchDownAndUp(300F, 300F))
-        //goView.perform(touchDownAndUp(500F, 500F))
-        //TODO this test will always pass, need to somehow actually test the result...
+        goView.perform(touchDownAndUp(1f, 1f))
+        goView.perform(touchDownAndUp(2f, 2f))
```

Comment:
Does this one test anything? I don't see the check to whether the stone is placed

Reply:
That's the thing, i really don't know how to test it exactly... With espresso you should be able to test if a bitmap was drawn but i'm not sure how. I guess you can see if the pixel at 1,1 turns black but then again it depends on the size of the view cause the emulator can be tiny. I will look into it